FirebaseDynamicLinks/CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+# v7.6.1
+- [added] Adding "utmParametersDictionary" api to FIRDynamicLink. (#7505)
```

Comment:
The issue number is preferable to the PR when both exist.

Reply:
done